libs/net/dal/Services/MediaTypeService.cs

```diff
@@ -20,5 +24,59 @@ public IEnumerable<MediaType> FindAll()
     {
         return this.Context.MediaTypes.OrderBy(a => a.SortOrder).ThenBy(a => a.Name).ToArray();
     }
+
+    public IPaged<MediaType> Find(MediaTypeFilter filter)
+    {
+        var query = this.Context.MediaTypes
+            .Include(c => c.Name)
```

Comment:
Include statements are for child entities (foreign keys).  They are not for property names like this.  I've fixed this in my PR.